etc/notebooks/sentiment_boilerplate_demo/sentiment_boilerplate_demo.ipynb

```diff
@@ -0,0 +1,797 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "urth": {
+     "dashboard": {
+      "hidden": true
+     }
+    }
+   },
+   "source": [
+    "#Sentiment Boilerplate\n",
+    "\n",
+    "This notebook provides a boilerplate for displaying and aggregating metrics about sentiment into a dashboard. In this notebook sentiment is defined as a positive or negative classification for some metric.\n",
```

Comment:
Would be nice to define boilerplate right at the top, something to the effect of "You can plug your own metrics into this notebook and then use it as a dashboard."